cirq/aqt/aqt_sampler.py

```diff
@@ -82,15 +82,20 @@ def _generate_json(
             json formatted string of the sequence
         """
 
-        seq_list: List[Tuple[str, float, List[int]]] = []
+        seq_list: Union[List[Tuple[str, float, List[int]]], List[
+            Tuple[str, float, float, List[int]]]] = []
         circuit = resolve_parameters(circuit, param_resolver)
         for op in circuit.all_operations():
             line_qubit = cast(Tuple[LineQubit], op.qubits)
             op = cast(ops.GateOperation, op)
             qubit_idx = [obj.x for obj in line_qubit]
             op_str = get_op_string(op)
             gate = cast(ops.EigenGate, op.gate)
-            seq_list.append((op_str, gate.exponent, qubit_idx))
+            if op_str == 'R':
+                seq_list.append((op_str, float(gate.exponent),
```

Comment:
Why is `float` called here and not in the `else`?

Reply:
This is an inconsistency connected to the failed mypy test. Working on it...

Reply:
Did we fix this issue yet?

Reply:
i think so: The parameter is now cast to float() in both cases.